db/memtable.cc

```diff
@@ -412,13 +412,28 @@ InternalIterator* MemTable::NewIterator(const ReadOptions& read_options,
   return new (mem) MemTableIterator(*this, read_options, arena);
 }
 
-InternalIterator* MemTable::NewRangeTombstoneIterator(
+FragmentedRangeTombstoneIterator* MemTable::NewRangeTombstoneIterator(
```

Comment:
Not related to this PR: I forget, were we also planning to make FragmentedRangeTombstoneIterator not a subclass of InternalIterator so we can get rid of some of its unused functionality?